onnxruntime/core/framework/finalize_session_state.cc

```diff
@@ -99,6 +99,8 @@ Status FinalizeSessionState(SessionState& session_state,
   session_state.CleanInitializedTensorsFromGraph();
 
   ORT_RETURN_IF_ERROR(session_state.CreateKernels(kernel_registry_manager));
+  ORT_RETURN_IF_ERROR(session_state.PrePackConstantTensors());
```

Comment:
Can we call this PrePackInitializedTensors to keep the naming consistent with SaveInitializedTensors and CleanInitializedTensorsFromGraph?